src/crypto/PSASpake2p.h

```diff
@@ -155,6 +155,7 @@ class PSASpake2p_P256_SHA256_HKDF_HMAC
     CHIP_ERROR GetKeys(SessionKeystore & keystore, HkdfKeyHandle & key);
 
 private:
+    psa_pake_cipher_suite_t mCipherSuite = PSA_PAKE_CIPHER_SUITE_INIT;
```

Comment:
Does it have to live as long as the operation? If not, shouldn't it be allocated on stack?

Reply:
Right.